### ISSUE TITLE
Update to 3.15

### ISSUE THREAD

**Issue by nickthomas6** [Esri/dojo-bootstrap-map-js#93]
Is there a task or vision to update esri from 3.13 to 3.15 and to swap from slurp over to a bower install or those artifacts?

**Comment by tomwayson** [comment]
Not really, I have recently worked on a project where we used Dojo-Bootstrap w/ JSAPI 3.15 and a bower build, so I can confirm that it all works.

Pull requests are welcome!

However, I personally don't start my projects w/ Dojo-Bootstrap these days. I find that unless it's essentially an entirely mobile-focused project, the pros of avoiding jQuery don't outweigh the cons.
Also, the Dojo-Bootstrap project does not (as far as I'm aware) have a plan to support the upcoming release of Bootstrap 4.

**Comment by nickthomas6** [comment]
Thanks Tom!  I see your point on that.  If I do have time I might work on a pull request.

Would you guys have other repos on here that I can look at to mimic my projects.  I've tried a couple of yeoman generators but didn't like the structure.  I liked this one for the use of Grunt, with building out dojo profiles and everything. Hoping to learn more at the developer summit!  Thanks again.
